docs/index.md

```diff
@@ -8,21 +8,21 @@ date: 2024-10-22
 ---
 # Welcome
 
-Welcome to the Department for Education Secure-by-Design development documentation.
+Welcome to the Department for Education (DfE)'s Secure by Design (SbD) site for development documentation.
 
 ## Overview
 
-This is where the SBD team in CISD are going to build out the plans for how CISD and the digital Portfolios will interact with the SBD process.
+The <abbr title= "Secure by Design">SbD</abbr> team sits within the Cyber and Information Security (CIS) division. We will use this site to outline and develop plans for how the <abbr title = "Cyber and Information Security">CIS</abbr> division and digital portfolios will interact with the <abbr title= "Secure by Design">SbD</abbr> process.
```

Comment:
We don't need to add `<abbr />` tags, this is what [the glossary](https://github.com/DFE-Digital/secure-by-design-docs/blob/main/includes/glossary.md?plain=1) is for. The site generator will convert everything in the glossary with the tags when it publishes.